mainnet-1/specs/fuse.json

```diff
@@ -0,0 +1,97 @@
+{
+    "proposal": {
+        "title": "Add Specs: Fuse",
+        "description": "Adding new specification support for relaying Fuse data on Lava",
+        "specs": [
+            {
+                "index": "FUSE",
+                "name": "fuse mainnet",
+                "enabled": true,
+                "imports": [
+                    "ETH1"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 34,
+                "blocks_in_finalization_proof": 0,
+                "average_block_time": 5040,
+                "allowed_block_lag_for_qos_sync": 2,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "15000000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "jsonrpc",
+                            "internal_path": "",
+                            "type": "POST",
+                            "add_on": ""
+                        },
+                        "apis": [],
+                        "headers": [],
+                        "inheritance_apis": [],
+                        "parse_directives": [],
+                        "verifications": [
+                            {
```

Comment:
add verifications for pruning and archive

Reply:
@Yaroms hey!

Can you please provide an example of this configuration or documentation which parameters I can specify. I looked into others specs and tbh not sure what are the right parameters.

Reply:
@Yaroms sorry, did changes. Please take a look.